rest-service/manager_rest/test/endpoints/test_deployment_labels_dependencies.py

```diff
@@ -1,613 +1,420 @@
             )
 
-    def test_upload_blueprint_with_valid_parent_id_on_dsl(self):
-        self.put_deployment('valid-id')
-        self.put_blueprint(
-            blueprint_id='bp1',
-            blueprint_file_name='blueprint_with_valid_parent_labels.yaml'
-        )
-
     def test_deployment_with_multiple_parent_labels(self):
-        self.put_deployment(deployment_id='parent_1',
-                            blueprint_id='blueprint_1')
-        self.put_deployment(deployment_id='parent_2',
-                            blueprint_id='blueprint_2')
-        self.put_deployment_with_labels(
-            [
-                {
-                    'csys-obj-parent': 'parent_1'
-                },
-                {
-                    'csys-obj-parent': 'parent_2'
-                }
+        parent1 = self._deployment(id='parent1')
+        parent2 = self._deployment(id='parent2')
+        self._deployment(id='dep')
+
+        self.client.deployments.set_attributes(
+            'dep',
+            labels=[
+                {'csys-obj-parent': 'parent1'},
+                {'csys-obj-parent': 'parent2'}
             ]
         )
-        deployment_1 = self.client.deployments.get('parent_1')
-        deployment_2 = self.client.deployments.get('parent_2')
-        self.assertEqual(deployment_1.sub_services_count, 1)
-        self.assertEqual(deployment_1.sub_environments_count, 0)
-        self.assertEqual(deployment_2.sub_services_count, 1)
-        self.assertEqual(deployment_2.sub_environments_count, 0)
-
-    def test_deployment_with_invalid_parent_label(self):
-        error_message = 'label `csys-obj-parent` that does not exist'
-        with self.assertRaisesRegex(CloudifyClientError, error_message):
-            self.put_deployment_with_labels(
-                [
-                    {
-                        'csys-obj-parent': 'notexist'
-                    }
-                ],
-                resource_id='invalid_label_dep'
-            )
+        assert parent1.sub_services_count == 1
+        assert parent1.sub_environments_count == 0
+        assert parent2.sub_services_count == 1
+        assert parent2.sub_environments_count == 0
 
     def test_deployment_with_valid_and_invalid_parent_labels(self):
-        self.put_deployment(deployment_id='parent_1')
-        error_message = 'label `csys-obj-parent` that does not exist'
-        with self.assertRaisesRegex(CloudifyClientError, error_message):
-            self.put_deployment_with_labels(
-                [
-                    {
-                        'csys-obj-parent': 'parent_1'
-                    },
-                    {
-                        'csys-obj-parent': 'notexist'
-                    }
-                ],
-                resource_id='invalid_label_dep'
+        parent = self._deployment(id='parent')
+        self._deployment(id='dep')
+        with self.assertRaisesRegex(CloudifyClientError, 'does not exist'):
```

Comment:
this will be later (not in this pr) replace with a check on a 404 exception class